packages/app/cypress/e2e/studio/studio.cy.ts

```diff
@@ -0,0 +1,242 @@
+import { launchStudio } from './helper'
```

Comment:
These test are failing when I run them locally, but are [passing in CI](https://app.circleci.com/pipelines/github/cypress-io/cypress/42523/workflows/a852ef89-7559-4da2-8bf6-738ac8949d51/jobs/1768275/parallel-runs/7?filterBy=ALL)? Is there something I'm missing when running these tests?

![Screen Shot 2022-08-26 at 9 55 31 AM](https://user-images.githubusercontent.com/25158820/186934382-9c757da8-8c6a-4b7e-9749-432e2b773698.png)

Reply:
Looking into this... this test is also failing for me but for a different reason

Reply:
Yeah the tests in this file are really flakey for me... I'm still looking into it. Seems like there's some situations where after `launchStudio` runs, we're not in studio mode. Probably a race condition of sorts

Reply:
Couldn't get this exact error. I tried a little commit https://github.com/cypress-io/cypress/pull/23544/commits/24d87183e024857ecd69946854cdc8fc83a69cd9 to reduce flake. These run reliably for me, and on CI :thinking:

Reply:
Yeah I'm still seeing them flake a bunch in `studio.cy.ts`. I've run them several times and can't get them all to pass at once 🤔 Must just be an issue with my machine. If they're running well in CI then I'd say this isn't a blocker